Makefile

```diff
@@ -157,8 +157,8 @@ code/lint:
 ##############################
 # CI                         #
 ##############################
-.PHONY: setup/travis
-setup/travis:
+.PHONY: setup/github
+setup/github:
```

Comment:
Technically these targets should be left as PHONY, because they don't produce a file with the target name. Doesn't really matter though.

Reply:
Oops, I misread the diff, nevermind :)